### PR TITLE
chore(gatsby-remark-copy-linked-files): Remove fs

### PR DESCRIPTION
**Pull request by kimbaudi**
## Description

remove redundant `const fs = require('fs')` module from [gatsby-remark-copy-linked-files](https://www.gatsbyjs.org/packages/gatsby-remark-copy-linked-files/) plugin since it already has `fs-extra` module

### CODE REVIEW

File: packages/gatsby-remark-copy-linked-files/src/index.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 const visit = require(`unist-util-visit`)
 const isRelativeUrl = require(`is-relative-url`)
-const fs = require(`fs`)
-const fsExtra = require(`fs-extra`)
+const fse = require(`fs-extra`)
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-copy-linked-files/src/index.js]:
```suggestion
const fsExtra = require(`fs-extra`)
```

File: packages/gatsby-remark-copy-linked-files/src/index.js [hunk 2]
```diff
@@ -150,7 +149,7 @@ module.exports = (
 
     if (!image.attr(`width`) || !image.attr(`height`)) {
       dimensions = imageSize.sync(
-        toArray(fs.readFileSync(imageNode.absolutePath))
+        toArray(fse.readFileSync(imageNode.absolutePath))
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-copy-linked-files/src/index.js]:
```suggestion
        toArray(fsExtra.readFileSync(imageNode.absolutePath))
```

File: packages/gatsby-remark-copy-linked-files/src/index.js [hunk 3]
```diff
@@ -299,10 +298,10 @@ module.exports = (
   return Promise.all(
     Array.from(filesToCopy, async ([linkPath, newFilePath]) => {
       // Don't copy anything if the file already exists at the location.
-      if (!fsExtra.existsSync(newFilePath)) {
+      if (!fse.existsSync(newFilePath)) {
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-copy-linked-files/src/index.js]:
```suggestion
      if (!fsExtra.existsSync(newFilePath)) {
```

File: packages/gatsby-remark-copy-linked-files/src/index.js [hunk 4]
```diff
@@ -299,10 +298,10 @@ module.exports = (
...
         try {
-          await fsExtra.ensureDir(path.dirname(newFilePath))
-          await fsExtra.copy(linkPath, newFilePath)
+          await fse.ensureDir(path.dirname(newFilePath))
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-copy-linked-files/src/index.js]:
```suggestion
          await fsExtra.ensureDir(path.dirname(newFilePath))
```

File: packages/gatsby-remark-copy-linked-files/src/index.js [hunk 5]
```diff
@@ -299,10 +298,10 @@ module.exports = (
...
+          await fse.copy(linkPath, newFilePath)
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-copy-linked-files/src/index.js]:
```suggestion
          await fsExtra.copy(linkPath, newFilePath)
```

File: packages/gatsby-remark-copy-linked-files/src/index.js (hunk 1 above)
Comment (kimbaudi) [reply packages/gatsby-remark-copy-linked-files/src/index.js]: done

File: packages/gatsby-remark-copy-linked-files/src/index.js (hunk 2 above)
Comment (kimbaudi) [reply packages/gatsby-remark-copy-linked-files/src/index.js]: done

**Comment by kimbaudi** [comment]
thank you! made suggested changes

**Pull request merged**